src/code42cli/cmds/users.py

```diff
@@ -252,13 +254,69 @@ def remove_alias(state, username, alias):
     _remove_cloud_alias(state.sdk, username, alias)
 
 
+@users.command()
+@click.argument("username")
+@click.argument("date", type=click.DateTime(formats=["%Y-%m-%d"]), required=False, metavar="DATE")
+@click.option("--clear", is_flag=True, help="Clears the current `start_date` value.")
+@sdk_options()
+def update_start_date(state, username, date, clear):
+    """Sets the `start_date` on a User's risk profile (useful for users on the New Hire Watchlist). 
+    Date format: %Y-%m-%d"""
+    if not date and not clear:
+        raise Code42CLIError("Must supply DATE argument if --clear is not used.")
+    if clear:
+        date = ""
+    user_id = _get_user(state.sdk, username)["userId"]
+    state.sdk.userriskprofile.update(user_id, start_date=date)
+    
+
+@users.command()
+@click.argument("username")
+@click.argument("date", type=click.DateTime(formats=["%Y-%m-%d"]), required=False)
+@click.option("--clear", is_flag=True, help="Clears the current `end_date` value.")
+@sdk_options()
+def update_departure_date(state, username, date, clear):
+    """Sets the `end_date` on a User's risk profile (useful for users on the Departing Employee 
```

Comment:
Super small - probably irrelevant but when I had him review my py42 docs John pointed out the me that the `Departing Employee` watchlist is now just titled `Departing` in the UI